tests/data/aws/sqs.py

```diff
@@ -19,3 +19,5 @@
         },
     ),
 ]
+
+LIST_SQS_QUEUE_URLS = [url for url, _ in GET_SQS_QUEUE_ATTRIBUTES]
```

Comment:
Is this required? I dont see it used in any of the tests. It could be used here but I don't think its worth creating a new variable https://github.com/cartography-cncf/cartography/pull/1619/files#diff-2e4ad934bded244df7809e07c8fa2e9b32106563cf2e655160d7355b26afaac2R101